src/common/options/mds-client.yaml.in

```diff
@@ -589,3 +589,13 @@ options:
   - mds_client
   flags:
   - runtime
+- name: client_fscrypt_as
+  type: bool
+  level: advanced
+  desc: Enable fscrypt access semantics
+  long_desc: Enable fscrypt access semantics for locked/unlocked states.
```

Comment:
What does this do exactly? I'm dubious about a client config to switch this on.

Reply:
I believe this is the option that lets a client look at the encrypted data (rather than returning standard fscrypt access errors). This is necessary to enable cloning of encrypted sub volumes.